examples/README.md

```diff
@@ -0,0 +1,19 @@
+# Examples
+
+This directory contains examples of artipie usages per each supported language and package type.
+
+Repository examples:
+
+| Type  | Status   |
+|---|---|
+| [Binary](./binary) | [![](https://github.com/artipie/artipie/workflows/Proof::binary/badge.svg)](./binary) |
+| [Maven](./maven) | [![](https://github.com/artipie/artipie/workflows/Proof::maven/badge.svg)](./maven) |
```

Comment:
@Sammers21 will each repository example folder have its own readme? This readme could contain information about each repository type we are removing from the main readme

Reply:
@olenagerasimova, yes, each example will have its own readme.